cypress/e2e/homepage.cy.js

```diff
@@ -19,5 +19,22 @@ describe('empty spec', () => {
       cy.get('.searchButton').click()
       cy.get('.addressErrorMsg').should('exist')
     })
+
+    it('should allow a user to log in', () => {
+      cy.get('.signInButton').click().wait(1000)
```

Comment:
We'll probably want to stub this network request at some point but I'm glad this is working.